<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="5" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-05T07:50:38 IST" name="ParallelMethodTesting" finished-at="2023-10-05T07:50:48 IST" duration-ms="10298">
    <groups>
    </groups>
    <test started-at="2023-10-05T07:50:38 IST" name="Test" finished-at="2023-10-05T07:50:48 IST" duration-ms="10298">
      <class name="com.obsquara.SevenRMart.testscript.ManageProdductTest">
        <test-method is-config="true" signature="intializeBrowser(java.lang.String)[pri:0, instance:com.obsquara.SevenRMart.testscript.ManageProdductTest@6f36c2f0]" started-at="2023-10-05T07:50:38 IST" name="intializeBrowser" finished-at="2023-10-05T07:50:47 IST" duration-ms="8728" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intializeBrowser -->
        <test-method signature="verifyUnableToaddNewIteminmanageproductpage()[pri:0, instance:com.obsquara.SevenRMart.testscript.ManageProdductTest@6f36c2f0]" started-at="2023-10-05T07:50:47 IST" name="verifyUnableToaddNewIteminmanageproductpage" finished-at="2023-10-05T07:50:47 IST" duration-ms="256" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.apache.poi.xssf.usermodel.XSSFWorkbook.getSheet(String)" because "Utilities.ExcelUtility.wb" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.apache.poi.xssf.usermodel.XSSFWorkbook.getSheet(String)" because "Utilities.ExcelUtility.wb" is null
at Utilities.ExcelUtility.getString(ExcelUtility.java:59)
at com.obsquara.SevenRMart.testscript.ManageProdductTest.verifyUnableToaddNewIteminmanageproductpage(ManageProdductTest.java:15)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUnableToaddNewIteminmanageproductpage -->
        <test-method is-config="true" signature="browserQuit(org.testng.ITestResult)[pri:0, instance:com.obsquara.SevenRMart.testscript.ManageProdductTest@6f36c2f0]" started-at="2023-10-05T07:50:47 IST" name="browserQuit" finished-at="2023-10-05T07:50:48 IST" duration-ms="1143" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUnableToaddNewIteminmanageproductpage status=FAILURE method=ManageProdductTest.verifyUnableToaddNewIteminmanageproductpage()[pri:0, instance:com.obsquara.SevenRMart.testscript.ManageProdductTest@6f36c2f0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserQuit -->
      </class> <!-- com.obsquara.SevenRMart.testscript.ManageProdductTest -->
      <class name="com.obsquara.SevenRMart.testscript.LoginTest">
        <test-method is-config="true" signature="intializeBrowser(java.lang.String)[pri:0, instance:com.obsquara.SevenRMart.testscript.LoginTest@1807f5a7]" started-at="2023-10-05T07:50:38 IST" name="intializeBrowser" finished-at="2023-10-05T07:50:47 IST" duration-ms="8820" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intializeBrowser -->
        <test-method signature="verifyUserIsAbleToLoginWithCorrectUserNameAndPassword()[pri:0, instance:com.obsquara.SevenRMart.testscript.LoginTest@1807f5a7]" started-at="2023-10-05T07:50:47 IST" name="verifyUserIsAbleToLoginWithCorrectUserNameAndPassword" finished-at="2023-10-05T07:50:47 IST" duration-ms="142" status="FAIL">
          <exception class="org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException">
            <message>
              <![CDATA[No valid entries or contents found, this is not a valid OOXML (Office Open XML) file]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException: No valid entries or contents found, this is not a valid OOXML (Office Open XML) file
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:145)
at org.apache.poi.openxml4j.util.ZipInputStreamZipEntrySource.<init>(ZipInputStreamZipEntrySource.java:49)
at org.apache.poi.openxml4j.opc.ZipPackage.<init>(ZipPackage.java:106)
at org.apache.poi.openxml4j.opc.OPCPackage.open(OPCPackage.java:307)
at org.apache.poi.ooxml.util.PackageHelper.open(PackageHelper.java:47)
at org.apache.poi.xssf.usermodel.XSSFWorkbook.<init>(XSSFWorkbook.java:309)
at Utilities.ExcelUtility.getString(ExcelUtility.java:54)
at com.obsquara.SevenRMart.testscript.LoginTest.verifyUserIsAbleToLoginWithCorrectUserNameAndPassword(LoginTest.java:18)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.util.zip.ZipException: Unexpected record signature: 0X82801A7F
at org.apache.commons.compress.archivers.zip.ZipArchiveInputStream.getNextZipEntry(ZipArchiveInputStream.java:260)
at org.apache.poi.openxml4j.util.ZipArchiveThresholdInputStream.getNextEntry(ZipArchiveThresholdInputStream.java:141)
... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- org.apache.poi.openxml4j.exceptions.NotOfficeXmlFileException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyUserIsAbleToLoginWithCorrectUserNameAndPassword -->
        <test-method is-config="true" signature="browserQuit(org.testng.ITestResult)[pri:0, instance:com.obsquara.SevenRMart.testscript.LoginTest@1807f5a7]" started-at="2023-10-05T07:50:47 IST" name="browserQuit" finished-at="2023-10-05T07:50:48 IST" duration-ms="1132" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyUserIsAbleToLoginWithCorrectUserNameAndPassword status=FAILURE method=LoginTest.verifyUserIsAbleToLoginWithCorrectUserNameAndPassword()[pri:0, instance:com.obsquara.SevenRMart.testscript.LoginTest@1807f5a7] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserQuit -->
      </class> <!-- com.obsquara.SevenRMart.testscript.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- ParallelMethodTesting -->
</testng-results>
